PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/AdminUsernameTextField.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.microsoft.azure.toolkit.intellij.mysql;
+
+import com.microsoft.azure.toolkit.intellij.common.ValidationDebouncedTextInput;
+import com.microsoft.azure.toolkit.lib.common.form.AzureValidationInfo;
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+
+public class AdminUsernameTextField extends ValidationDebouncedTextInput {
+
+    private static final int LENGTH_MIN = 1;
+    private static final int LENGTH_MAX = 16;
+    private static final String LENGTH_INVALILD_MESSAGE = "Admin username must be at least 1 characters and at most 16 characters.";
+    private static final String ALPHANUMBERIC_INVALID_MESSAGE = "Admin username must only contain characters and numbers.";
+    private static final String[] FIXED_NAMES_INVALID = new String[]{"azure_superuser", "admin", "administrator", "root", "guest", "public"};
+    private static final String FIXED_NAMES_INVALID_MESSAGE =
+            "Admin login name cannot be 'azure_superuser', 'admin', 'administrator', 'root', 'guest' or 'public'.";
+
+    @Getter
+    @Setter
+    private boolean passwordInitialized;
```

Comment:
where to change passwordInitialized?

Reply:
The create dialog of mysql. will included in the next PR.